plugins/outputs/opentelemetry/sample.conf

```
@@ -24,6 +24,17 @@
   ## Supports: "gzip", "none"
   # compression = "gzip"
 
+  ## Configuration options for the Coralogix dialect
+  ## Enable the following section of you use this plugin with a Coralogix endpoint
+  # [outputs.opentelemetry.coralogix]
+  #   ## Your Coralogix private key (required).
+  #   ## Please note that this is sensitive data!
+  #   private_key: "your_coralogix_key"
+  #
+  #   ## Application and subsystem names for the metrics (required)
+  #   application: "$NAMESPACE"
+  #   subsystem: "$HOSTNAME"
```

We are usually using `=` in TOML, so I suggest to keep it consistent...
```suggestion
  #   ## Your Coralogix private key (required).
  #   ## Please note that this is sensitive data!
  #   private_key = "your_coralogix_key"
  #
  #   ## Application and subsystem names for the metrics (required)
  #   application = "$NAMESPACE"
  #   subsystem = "$HOSTNAME"
```